src/utils/curriculum/features.ts

```diff
@@ -32,14 +24,7 @@ export function getUnitFeatures(unit?: Unit | null) {
     return;
   }
 
-  // HACK: Swimming primary isn't yet published so we're hacking in some secondary units and giving them the overrides
-  if (
-    isSwimmingHackEnabled() &&
-    [
-      "health-and-wellbeing-hiit-and-couch-to-5k-team-challenges-to-develop-fitness",
-      "sport-psychology-skill-and-ability",
-    ].includes(unit.slug)
-  ) {
+  if (unit.features?.pe_swimming) {
```

Comment:
Excuse my ignorance, but does this check that `pe_swimming` is true, or that the key exists?

Reply:
(I looked this up myself, all good!)